plugins/org.jkiss.dbeaver.ext.dm/src/org/jkiss/dbeaver/ext/dm/edit/DMSequenceManager.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.dm.edit;
+
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.ext.dm.model.DMSchema;
+import org.jkiss.dbeaver.ext.generic.model.GenericSequence;
+import org.jkiss.dbeaver.ext.generic.model.GenericStructContainer;
+import org.jkiss.dbeaver.model.DBPDataSource;
+import org.jkiss.dbeaver.model.DBPEvaluationContext;
+import org.jkiss.dbeaver.model.edit.DBECommandContext;
+import org.jkiss.dbeaver.model.edit.DBEPersistAction;
+import org.jkiss.dbeaver.model.exec.DBCExecutionContext;
+import org.jkiss.dbeaver.model.impl.edit.SQLDatabasePersistAction;
+import org.jkiss.dbeaver.model.impl.sql.edit.SQLObjectEditor;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.dbeaver.model.struct.DBSObject;
+import org.jkiss.dbeaver.model.struct.cache.DBSObjectCache;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author Shengkai Bai
+ */
+public class DMSequenceManager extends SQLObjectEditor<GenericSequence, DMSchema> {
+    @Override
+    public long getMakerOptions(DBPDataSource dataSource) {
+        return FEATURE_EDITOR_ON_CREATE;
+    }
+
+    @Override
+    public DBSObjectCache<? extends DBSObject, GenericSequence> getObjectsCache(GenericSequence object) {
+        return null;
+    }
+
+    @Override
+    protected GenericSequence createDatabaseObject(DBRProgressMonitor monitor, DBECommandContext context, Object container, Object copyFrom, Map<String, Object> options) throws DBException {
+        return new GenericSequence((GenericStructContainer) container, "NEW_SEQUENCE", null, 0, 1, Long.MAX_VALUE, 1);
```

Comment:
Please copy logic from this class:

https://github.com/dbeaver/dbeaver/blob/359a137b918c29ab5953493e772ced451c97749b/plugins/org.jkiss.dbeaver.ext.vertica/src/org/jkiss/dbeaver/ext/vertica/edit/VerticaSequenceManager.java#L52-L63

To avoid duplicating new object names.